README.md

```diff
@@ -111,6 +111,36 @@ kubectl get pods -n codewind
 If successful, you should see the codewind-operator pod running and ready for work.
 
 
+## Persistent storage requirements
+
+Keycloak and Codewind pods have storage requirements. Both require available PersistentStorage to be configured and available before you attempt to deploy each service.
+
+Each Keycloak instance requires by default:
+
+- 1Gi capacity
+- access mode of RWO (ReadWriteOnly)
+
+Each Codewind instance requires by default:
+
+- 10Gi capacity
+- access mode of RWX (ReadWriteMany)
+
```

Comment:
access => Access (initial caps) to both above instances, so becomes:
- Access mode...